packages/wasm-dpp/scripts/build-wasm.sh

```diff
@@ -25,10 +25,10 @@ if ! [[ -d $OUTPUT_DIR ]];  then
   mkdir -p "$OUTPUT_DIR"
 fi
 
-if ! [ -x "$(command -v wasm-bindgen)" ]; then
-    echo 'Wasm-bindgen CLI is not installed. Installing';
-    cargo install -f wasm-bindgen-cli
-fi
+#if ! [ -x "$(command -v wasm-bindgen)" ]; then
+#    echo 'Wasm-bindgen CLI is not installed. Installing';
+    cargo install -f wasm-bindgen-cli@0.2.84
```

Comment:
Hm. Is it going to be re-installed on every script run? Or will cargo detect that this version is already installed and just wont do anything?

Reply:
It is going to. The reason is that no matter what I did if I didn't forcibly reinstall it on every run, something broke in the CI. I mentioned it on yesterday's standup

Reply:
Agreed on fixing this in the next PR